pkg/crd/crdloader/crdloader.go

```diff
@@ -138,7 +137,13 @@ func isMatch(group, version, kind string, crd apiextensions.CustomResourceDefini
 		}
 	}
 	if version != "" {
-		if k8s.GetVersionFromCRD(&crd) != version {
+		foundVersion := false
```

Comment:
Nit: Probably easier for readability and reusability if it is a util function (e.g. `ContainsVersionInCRD(crd, version)`) under k8s package.